docs/prefix-and-ip-target.md

```diff
@@ -9,26 +9,26 @@ When a new ENI is allocated, IPAMD will allocate either 1 prefix or number of pr
\ No newline at end of file
+| Instance type | `WARM_PREFIX_TARGET`| `WARM_IP_TARGET`| `MINIMUM_IP_TARGET` | Pods | ENIs | Pod per ENIs | Attached Prefixes | Unused Prefixes | Prefixes per ENI | Unused IPs|
+|---------------|:-------------------:|:---------------:|:-------------------:|:----:|:----:|:------------:|:-----------------:|:---------------:|:----------------:|:---------:|
+| t3.small      |         1           |         -       |          -          |   0  |  1   |      0       |        1          |      1          |      1           |    16     |
+| t3.small      |         1           |         -       |          -          |   5  |  3   |    1,2,2     |        4          |      1          |    2,1,1         |    59     |
+| t3.small      |         1           |         -       |          -          |  17  |  1   |     17       |        3          |      1          |      3           |    31     |
+|               |                     |                 |                     |      |      |              |                   |                 |                  |           |
+| t3.small      |         -           |         1       |          1          |   0  |  1   |      0       |        1          |      1          |      1           |    16     |
+| t3.small      |         -           |         1       |          1          |   5  |  3   |    1,2,2     |        3          |      0          |    1,1,1         |    43     |
+| t3.small      |         -           |         1       |          1          |  17  |  1   |     17       |        2          |      0          |      2           |    15     |
+|               |                     |                 |                     |      |      |              |                   |                 |                  |           |
+| t3.small      |         -           |         2       |         10          |   0  |  1   |      0       |        1          |      1          |      1           |    16     |
+| t3.small      |         -           |         2       |         10          |   5  |  3   |    1,2,2     |        3          |      0          |    1,1,1         |    43     |
+| t3.small      |         -           |         2       |         10          |  17  |  1   |     17       |        2          |      0          |      2           |    15     |
+|               |                     |                 |                     |      |      |              |                   |                 |                  |           |
+| p3dn.24xlarge |         1           |         -       |          -          |   0  |  1   |      0       |        1          |      1          |      1           |    16     |
+| p3dn.24xlarge |         1           |         -       |          -          |   3  |  2   |    3,0       |        3          |      1          |    2,0           |    29     |
```

Comment:
Should this be 2,0 -> 2,1 given attached prefix is 3?

Reply:
Actually attached prefix should be 2 here..